<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog1</class>
 <widget class="QWidget" name="dialog1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lowEdit_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lowEdit_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>1250</width>
     <height>750</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1250</width>
     <height>750</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1250</width>
     <height>750</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>filter for stationID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lowEdit_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>620</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>load file..</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="upEdit_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lowEdit_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>filter for payType</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lowEdit_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>filter for deviceID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="upEdit_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lowEdit_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>filter for status</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>filter for lineID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>filter for userID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="upEdit_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="upEdit_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="upEdit_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="upEdit_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>670</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>filter for time</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="lowEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2019</year>
     <month>1</month>
     <day>7</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd hh:mm:ss</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="upEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="date">
    <date>
     <year>2019</year>
     <month>1</month>
     <day>7</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd hh:mm:ss</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
